Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp

```diff
@@ -180,7 +180,8 @@ static ErrorOr<Optional<String>> generate_enum_initializer_for(StringView proper
         { "button_style"sv, "Gfx::ButtonStyle"sv },
         { "opportunistic_resizee"sv, "GUI::Splitter::OpportunisticResizee"sv },
         { "checkbox_position"sv, "GUI::CheckBox::CheckBoxPosition"sv },
-        { "button_style"sv, "Gfx::ButtonStyle"sv }
+        { "button_style"sv, "Gfx::ButtonStyle"sv },
+        { "orientation"sv, "Gfx::Orientation"sv }
```

Comment:
This should work fine with a trailing `,`. That way, we don't have to touch two lines each time that we extend the list.